### DISCUSSION TITLE
Navneendring og rekkefølge på fonter i Figmabiblioteket

### DISCUSSION THREAD

**Discussion by Stormoen** [fremtind/jokul#3334] [Annet]
1) Kan "bold" døpes om til "body bold"? Ordene er så like at jeg trykker feil så ofte  Da følges også samme prinsipp som "small" - "small bold".
2) Tipper de fleste bruker de minste fontstilene mest i hverdagen. Kunne man vært så gæren å snudd lista, og starta med de minste stilene innenfor desktop? Da hadde man sluppet scroll til body.

![8dba0dc8-6cd8-4ab6-86bc-076654e0d9f4](https://user-images.githubusercontent.com/64519175/209103185-b21304b1-ffd2-410d-8cb2-d3e01abd5b02.jpg)

**Comment by wkillerud** [comment]
Det gir mening! Laget et issue på det, #3337.
